x-pack/solutions/security/plugins/security_solution/public/asset_inventory/routes.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { SecuritySubPluginRoutes } from '../app/types';
+import { SecurityPageName } from '../app/types';
+import { ASSET_INVENTORY_PATH } from '../../common/constants';
+import { PluginTemplateWrapper } from '../common/components/plugin_template_wrapper';
+import { SecurityRoutePageWrapper } from '../common/components/security_route_page_wrapper';
+import { ExperimentalFeaturesService } from '../common/experimental_features_service';
+import { AssetInventoryContainer } from './pages';
+
+export const AssetInventoryRoutes = () => (
+  <PluginTemplateWrapper>
+    <SecurityRoutePageWrapper pageName={SecurityPageName.assetInventory}>
+      <AssetInventoryContainer />
+    </SecurityRoutePageWrapper>
+  </PluginTemplateWrapper>
+);
+
+export const routes: SecuritySubPluginRoutes = [
+  {
+    path: ExperimentalFeaturesService.get().assetInventoryStoreEnabled ? ASSET_INVENTORY_PATH : [],
```

Comment:
I think we can just put ```path: ASSET_INVENTORY_PATH,``` here,
I believe the feature flag should have been taken into account via `experimentalKey` in `x-pack/solutions/security/plugins/security_solution/public/asset_inventory/links.ts`

Reply:
@angorayc Doesn't `experimentalKey` only affect the link's visibility based on the flag? Here we wanted to prevent users from accessing the plugin's page by manually entering the URL. If you confirm we still don't need to do that, I can remove it ofc

Reply:
@semd I forgot auto-merge was enabled, but will take care of that for sure!